components/Header.tsx

```diff
@@ -16,6 +17,10 @@ export default function Header() {
     setOpen(false);
   };
 
+  const handleNotification = () => {
+    setNotificationOpen(!notificationOpen);
```

Comment:
Here we can add a cookie and a clearInterval in order to shown the automatic notification just one time. The cookie need a long term date, a month will be good